### ISSUE TITLE
Firebase uploaded audio file does not work in ios

### ISSUE THREAD

**Issue by itxhamzajaved** [zmxv/react-native-sound#806]
Here is the file url: https://firebasestorage.googleapis.com/v0/b/hisnul-muslim-plus.appspot.com/o/AudioFiles%2F067jsjCHao6HfdoOQtShrToGTHN4?alt=media&token=ae421f15-ed7c-4f9a-acdc-e39f908fd05f
It gives error when we try to load this url in ios but working fine in android

**Comment by chandu-abhi** [comment]
Yes storage links are not working, but you can use "React-native-blob-utils", using this lib,  pass storage url and download that file locally and pass that locally downloaded path to sound library. that will work.

**Comment by BraveEvidence** [comment]
This will help https://www.youtube.com/watch?v=vVI7ZAZq5e0
